src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -32,7 +32,7 @@ export class AddBulkConcepts extends Component {
     const dictionaryName = localStorage.getItem('dictionaryName');
     const { cielConcepts } = this.props;
     return (
-      <div className="container add-bulk-concepts">
```

Comment:
why did you remove the container class?

Reply:
I have removed the container class since it was aligning that 'div' to the center by default which should not be the case here.